frontend/src/scenes/billing/billingLogic.tsx

```diff
@@ -85,6 +87,54 @@ const parseBillingResponse = (data: Partial<BillingType>): BillingType => {
     return data as BillingType
 }
 
+const storeBillingAlertDismissal = (
```

Comment:
IIRC having functions in the logic can cause weird issues so just make sure you retest it all before merging.